app/views/vm_common/console_webmks.html.haml

```diff
@@ -28,6 +28,11 @@
         $('#ctrlaltdel').on('click', function() {
           wmks.sendCAD();
         });
+
+        $('#keymap').on('change', function() {
+          console.log(this.value);
```

Comment:
Is the log here intentional?

Reply:
yes @mzazrivec testing your review skills :wink: